src/js/stores/SessionStore.js

```diff
@@ -11,7 +11,7 @@ var _session = {
 	emSize: 10,
 	width: 640,
 	timerOn: (localStorage.hasOwnProperty("model") === true),
-	nowOffset: getTZOffset(now),
+	nowOffset: now.getTimezoneOffset(),
```

Comment:
Ok after further research I am sure this is wrong. Is there any way to exclude a single file from a commit... or should I just add a new commit which fixes, and then add it to the PR?